c7n/resources/rds.py

```diff
@@ -1568,3 +1568,105 @@ def process(self, resources, event=None):
                     results.append(resource)
                     break
         return results
+
+
+@actions.register('modify-db')
+class ModifyDb(BaseAction):
+    """Modifies an RDS instance based on specified parameter
+    using ModifyDbInstance.
+
+    'Update' is an array with with key value pairs that should be set to
+    the keyword and value you wish to modify.
+    'Immediate" determines whether the modification is applied immediately
+    or not. If 'immediate' is not specified, default is false.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: disable-rds-deletion-protection
+                resource: rds
+                filters:
+                  - DeletionProtection: true
+                  - PubliclyAccessible: true
+                actions:
+                  - type: modify-db
+                    update:
+                        - keyword: 'DeletionProtection'
+                            value: false
```

Comment:
the indentation in the example looks off

Reply:
fixed